packit_service/models.py

```diff
@@ -956,6 +956,7 @@ class SRPMBuildModel(Base):
     copr_builds = relationship("CoprBuildModel", back_populates="srpm_build")
     koji_builds = relationship("KojiBuildModel", back_populates="srpm_build")
     build_submitted_time = Column(DateTime, default=datetime.utcnow)
+    url = Column(Text)
```

Comment:
It's not so important, but would be nice if we can fill those values during the database upgrade for older SRPMs.

Reply:
oh my, that would be so cool

Reply:
And not so hard I hope.

Reply:
with @lachmanfrantisek we agreed on not setting URLs for older SRPMs, my main reasons for not doing so is deletion of projects and related files after 60 days in COPR which would result in keeping 404 links in DB

We probably should also look into deletion of them after they're expired (looking at the `CoprBuildModel` there might be more expired URLs that we keep in database)